docs/content/getting-started.md

```diff
@@ -83,43 +87,69 @@ If you prefer to configure the service with commandline flags you can see the av
 
 --debug-token
 : Token to grant metrics access, empty default value
+#### Health
 
 --debug-pprof
 : Enable pprof debugging, defaults to `false`
 
 --http-addr
 : Address to bind http server, defaults to `0.0.0.0:8380`
+### <a name="configuration-files"></a>Configuration Files
 
 --http-root
 : Root path for http endpoint, defaults to `/`
+Configuration files can be written in either `JSON` or `YAML` format. Configuration files are automatically loaded if they're placed in the following directories:
 
 --asset-path
 : Path to custom assets, empty default value
+- `/etc/ocis/hello.yml`
+- `${HOME}/.ocis/hello.yml`; or
+- `$(pwd)/config/hello.yml`
 
-#### Health
+Full example configurations are available [in our repository](repo), which include all available options and default values.
 
 --debug-addr
 : Address to debug endpoint, defaults to `0.0.0.0:8390`
+## Usage
 
-### Configuration file
+The program supports a number of sub-commands; these are:
 
-So far we support the file formats `JSON` and `YAML`, if you want to get a full example configuration just take a look at [our repository](repo), there you can always see the latest configuration format. These example configurations include all available options and the default values. The configuration file will be automatically loaded if it's placed at `/etc/ocis/hello.yml`, `${HOME}/.ocis/hello.yml` or `$(pwd)/config/hello.yml`.
+- [server](#command-server)
+- [health](#command-health)
 
-## Usage
+**Note:** You can run `ocis-hello `--help`` for the available flags (as in the example below), and "ocis-hello [command] `--help`" for more information about a sub-command.
 
-The program provides a few sub-commands on execution. The available configuration methods have already been mentioned above. Generally you can always see a formated help output if you execute the binary via `ocis-hello --help`.
+{{< highlight txt >}}
```

Comment:
I don't like to print the whole help output here, it will never stay properly in sync.